web/pwa.json

```diff
@@ -1,7 +1,7 @@
 {
 	"lang": "en",
 	"dir": "ltr",
-	"name": "Cimbar Encoder",
+	"name": "cimbar.org Send",
```

Comment:
Normalizing the naming a bit. The user facing tools will be called "send/receive", while the code itself will talk about things like encoding and decoding.